packages/systemd/9001-move-stateful-paths-to-ephemeral-storage.patch

```diff
@@ -1,45 +0,0 @@
-From 4f14d52fb6951f3870bfbe6789471cd75a87c341 Mon Sep 17 00:00:00 2001
-From: Ben Cressey <bcressey@amazon.com>
-Date: Sun, 15 Sep 2019 00:21:26 +0000
-Subject: [PATCH 9001/9005] move stateful paths to ephemeral storage
-
-We reserve most of /var for persistent local storage controlled by
-the administrator, and want to avoid depending on it for our own
-services.
-
-Signed-off-by: Ben Cressey <bcressey@amazon.com>
----
- src/libsystemd/sd-path/sd-path.c | 8 ++++----
- 1 file changed, 4 insertions(+), 4 deletions(-)
-
-diff --git a/src/libsystemd/sd-path/sd-path.c b/src/libsystemd/sd-path/sd-path.c
-index 95d6551..ac3e6fc 100644
---- a/src/libsystemd/sd-path/sd-path.c
-+++ b/src/libsystemd/sd-path/sd-path.c
-@@ -251,19 +251,19 @@ static int get_path(uint64_t type, char **buffer, const char **ret) {
-                 return 0;
- 
-         case SD_PATH_SYSTEM_STATE_PRIVATE:
--                *ret = "/var/lib";
-+                *ret = "/run/lib";
-                 return 0;
- 
-         case SD_PATH_SYSTEM_STATE_LOGS:
--                *ret = "/var/log";
-+                *ret = "/run/log";
-                 return 0;
- 
-         case SD_PATH_SYSTEM_STATE_CACHE:
```

Comment:
Is there an expected set of use cases that suggest we keep `/var/cache` around? I'm wondering if reconfigurations and upgrades may have implicit conflicts with data under this path that we can avoid by *not* persisting `/var/cache`.

Reply:
migrator uses `/var/cache/bottlerocket-metadata` after reboot.

Reply:
We don't have any uses of `/var/cache` today outside of updog and migrator.

We have a `/var/cache/private` that's created for use with the `CacheDirectory` directive in unit files, but we don't use that feature.